### ISSUE TITLE
Water appearance when cup is empty

### ISSUE THREAD

**Issue by marlitas** [phetsims/mean-share-and-balance#14]
Currently when the water in the 3D cup is dragged all the way down and reaches 0, we still see the top layer of water as if there is remaining liquid or a puddle in the cup. 

<img width="244" alt="Screen Shot 2022-05-10 at 10 00 11 AM" src="https://user-images.githubusercontent.com/42476338/167672676-179f2994-baf9-4aaf-bf6f-425b1ffb1475.png">

- Should the water completely disappear when the level reaches 0, only showing an empty cup?
- If so how do we want the water to disappear? some examples: jump from present to not present, an opacity change as it gets closer to the bottom. 

@samreid may have more ideas on what is possible here.

**Comment by samreid** [comment]
Yes, perhaps when the water level is low (like `< 1E-6`?) the water should immediately disappear (don't render the ellipse).

**Comment by amanda-phet** [comment]
@samreid that idea sounds reasonable to me! I agree that at a certain point we need to not show the blue ellipse.
